tweaktweak.html

```diff
@@ -0,0 +1,371 @@
+                            <img style="vertical-align:middle" src="icons/notifications-blue.svg">
+                            <span>Notifications</span>
+                        </div>
+
+                    </li>
+                    <li>
+                        <div id="messages">
+                            <img style="vertical-align:middle" src="icons/messages-blue.svg">
+                            <span>Messages</span>
+                        </div>
+
+                    </li>
+                    <li>
+                        <div id="bookmarks">
+                            <img style="vertical-align:middle" src="icons/bookmarks-blue.svg">
+                            <span>Bookmarks</span>
+                        </div>
+
+                    </li>
+                    <li>
+                        <div id="lists">
+                            <img style="vertical-align:middle" src="icons/lists-blue.svg">
+                            <span>Lists</span>
+                        </div>
+
+                    </li>
+                    <li>
+                        <div id="profile">
+                            <img style="vertical-align:middle" src="icons/profile-blue.svg">
+                            <span>Profile</span>
+                        </div>
+
+                    </li>
+                    <li>
+                        <div id="more11">
+                            <img style="vertical-align:middle" src="icons/more-blue.svg">
+                            <span>More</span>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+            <div class="blank-list">
+                <ul>
+                    <li>
+                        <div id="one"></div>
+                    </li>
+                    <li>
+                        <div id="two"></div>
+                    </li>
+                    <li>
+                        <div id="three"></div>
+                    </li>
+                    <li>
+                        <div id="four"></div>
+                    </li>
+                    <li>
+                        <div id="five"></div>
+                    </li>
+                    <li>
+                        <div id="six"></div>
+                    </li>
+                    <li>
+                        <div id="seven"></div>
+                    </li>
+                    <li>
+                        <div id="eight"></div>
+                    </li>
+                </ul>
+            </div>
+
+
+
+
+        </div>
+
+
+
+
+
+
+        <div class="tweak">Tweak</div>
+        <div class="bottom-left">
+            <div id="bottom-profile"><img src="dimages/profile.png"></div>
+            <div id="bottom-info">
+                <h4>Daniel McCarthy</h4>
+                <p>@Dan1212</p>
+            </div>
+            <div id="three-dots"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
```

Comment:
It's a good practice to include SVGs in an `<img>` tag such as `<img src="./icons/my-icon.svg" />`. This way we reduce the number of lines in our html and make it more readable :)

Reply:
Ah I can see that you've done that lower down `<img src="icons/star.svg" />`.